sys-block/rts_pstor/rts_pstor-1.10_p20160103.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit linux-mod
+
+GIT_COMMIT="c8313abffe083ac63bf76d2cc90d3edf5b2d1188"
+
+DESCRIPTION="PCIE RTS5209 card reader driver for Linux"
+HOMEPAGE="https://github.com/gexplorer/RTS5209-linux-driver"
+SRC_URI="https://github.com/gexplorer/RTS5209-linux-driver/archive/${GIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND=""
```

Comment:
You can remove them if they're really empty.

Reply:
Ok.